bin/push_to_wordpress_org.sh

```diff
@@ -13,7 +13,7 @@ svn co \
 
 if [ -d "wp_org/tags/$PLUGIN_VERSION" ]; then
   echo "Version exist. Exiting."
-  exit 1
+  exit 12
```

Comment:
@gilbertk88 Why are you setting exit code 12?

Reply:
I rolled it back to 1

Reply:
Gilbert,
Kindly reach me. I had to check the web for a means to reach you with respect to the simplelender WP plugin. My email is : leyini2010@gmail.com. Apologies if am sending this in the wrong environment.